server/simple_server.go

```diff
@@ -53,7 +50,6 @@ func NewSimpleServer(cfg *config.Server) *SimpleServer {
 		cfg:      cfg,
 		exit:     make(chan chan error),
 		monitor:  NewActivityMonitor(),
-		ctx:      netContext.Background(),
```

Comment:
I think we can leave a context object on the `SimpleServer` in case the server needs it for background processing or configurations for accessing services. We just need not pass it into the request middleware like before. (This is something required for running Gizmo on App Engine 'managed VMs' https://github.com/NYTimes/gizmo/blob/appengine/server/simple_server.go#L52-L55)